content/blog/posts/this-is-how-we-watchdog.md

```diff
@@ -0,0 +1,36 @@
+---
+title: This is how we watchdog
+date: 2020-12-21
+author: Eva Ruth Moravec
+---
+We frequently mention that we “watchdog” the data that we work with as much as possible, but I thought I’d peel that back a bit. What do TJI’s oversight efforts look like? 
+
+Each month, I get new data from the Texas Office of the Attorney General reflecting the previous month's reported deaths in custody and shootings of and by law enforcement officer. I add this new data to our master data sets. Overnight, bots look for changes to the data sets, which are then pushed through our processing pipeline. 
+
+The data-entry process is still quite manual, and I often find errors in the submitted reports – everything from misspellings and wrong names to transposed dates of death that result in negative ages and narratives obviously pulled from a different report. I note these, as well as reports that are missing, meaning it hasn't been filed within the required 30-day time period. **I recently went through this process and wanted to document and narrate what this effort looks like.** 
+
+First observation: There were quite a few custodial deaths in November – 144 *new* deaths for that month alone, when we usually see around 90. Next, I noted a few missing custodial death reports: 
+
+\- 11/5 fatal shooting of Reginald Alexander Jr. in [Dallas](https://dpdbeat.com/2020/11/10/deputy-chief-reuben-ramirez-discusses-officer-involved-shooting-with-reginald-alexander-jr/); 
+
+\- 11/18 fatal shooting of Pedro Martinez Jr. on the [Tyler Junior College](https://tylerpaper.com/news/local/breaking-tyler-man-dies-after-being-shot-by-tjc-officer/article_2fb73428-2a05-11eb-8221-6b9889e74ced.html) campus; 
+
+\- 10/3 fatal shooting of Jonathan Price by [Wolfe City police](https://www.fox4news.com/news/hunt-county-grand-jury-reviewing-wolfe-city-officers-murder-case); and 
+
+\- 10/9 fatal shooting of Ariel Esau Lujan by [Houston police](https://cityofhouston.news/investigation-into-fatal-shooting-at-6229-almeda-genoa-road/). 
+
+Then, I noted that there were *two* reports filed by separate people at the Texas Department of Criminal Justice for the 11/15 death of Omar Rojas: 
+
+![](https://res.cloudinary.com/texas-justice-initiative/image/upload/v1608508768/OmarRojas_OAGSite.png)
+
+And finally, the officer-involved shooting report for Reginald Alexander Jr. had been submitted by Dallas police, though media reported that he was shot by officers from another agency. 
```

Comment:
note presence of +/- box now
```suggestion
And finally, the officer-involved shooting report for Reginald Alexander Jr. had been submitted by Dallas police, even though media reported that he was shot by officers from another agency. 
```

Reply:
[Screengrab.zip](https://github.com/texas-justice-initiative/website-nextjs/files/5732368/Screengrab.zip)
not sure if this is the most efficient way to share/upload the video of what I am seeing but I hope it works! @nholden @hongsups

Reply:
for others reading - the comments above make sense if you watch the attached video

Reply:
Thanks for the video! I think it's because you were on the "commits" tab on the top of the page. If you view changes from the "files changed" tab, you should always be able to see the suggestion icon. (Sorry this isn't more intuitive!)

Reply:
ahh indeed!! thanks so much